src/packages/@ncigdc/modern_components/Lolliplot/index.js

```diff
@@ -0,0 +1,25 @@
+import { compose, withPropsOnChange } from 'recompose';
+import Component from './LolliplotWrapper';
+import createRendererSsm from './SsmLolliplot.relay';
+import createRendererGene from './GeneLolliplot.relay';
+
+const GeneLolliplot = createRendererGene(Component);
+
+const GeneLolliplotWrapper = compose(
```

Comment:
its' only relevant for the Ssm Page, the variable is poorly named I guess, It's like GeneLolliplotRendererForSsm